protocols/identify/Cargo.toml

```diff
@@ -24,6 +23,7 @@ wasm-timer = "0.2"
 unsigned-varint = { version = "0.2.3", features = ["futures-codec"] }
 
 [dev-dependencies]
+async-std = "1.0"
```

Comment:
We should probably import `async-std` everywhere as `1` instead of `1.x`. According to their [stability policy](https://book.async.rs/overview/stability-guarantees.html) changes in minor and patch versions are backwards compatible.

Reply:
`async-std = "1"` and `async-std = "1.0"` should do exactly the same thing. Cargo will pull `async-std 1.1` even if we say `1.0`.